packages/prisma/migrations/20231215054122_organization_metadata/migration.sql

```diff
@@ -0,0 +1,45 @@
+-- AlterTable
+ALTER TABLE "Team" ADD COLUMN     "isOrganization" BOOLEAN NOT NULL DEFAULT false;
+
+-- We can now add a unique constraint between slug and isOrganization
+CREATE UNIQUE INDEX "Team_slug_isOrganization_key" ON "Team"("slug", "isOrganization");
+
+-- CreateTable
+CREATE TABLE "OrganizationSettings" (
+    "id" SERIAL NOT NULL,
+    "teamId" INTEGER NOT NULL,
+    "isOrganizationConfigured" BOOLEAN NOT NULL DEFAULT false,
+    "isOrganizationVerified" BOOLEAN NOT NULL DEFAULT false,
+    "orgAutoAcceptEmail" TEXT NOT NULL,
+
+    CONSTRAINT "OrganizationSettings_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "OrganizationSettings_teamId_key" ON "OrganizationSettings"("teamId");
+
+-- AddForeignKey
+ALTER TABLE "OrganizationSettings" ADD CONSTRAINT "OrganizationSettings_teamId_fkey" FOREIGN KEY ("teamId") REFERENCES "Team"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- Set team field to notify if it is an organization -> Easier than metadata.parse(X).isOrganization
+UPDATE "Team"
+    SET "isOrganization" = (metadata ->> 'isOrganization')::BOOLEAN
+WHERE
+    metadata ->> 'isOrganization' = 'true';
+
+-- Insert data into org settings
+INSERT INTO "OrganizationSettings" ("teamId", "isOrganizationConfigured", "orgAutoAcceptEmail", "isOrganizationVerified")
+SELECT
+	t.id,
+	(t.metadata ->> 'isOrganizationConfigured')::BOOLEAN AS "isOrganizationConfigured",
+	t.metadata ->> 'orgAutoAcceptEmail' AS "orgAutoAcceptEmail",
+	(t.metadata ->> 'isOrganizationVerified')::BOOLEAN AS "isOrganizationVerified"
+FROM (
+	SELECT
+		id,
+		metadata
+	FROM
+		"Team"
+	WHERE
+		metadata ->> 'isOrganization' = 'true'
+) AS t;
```

Comment:
important: 

Here we are just updating a table - we are not removing or deleting this data from metadata FOR NOW. 

Once we have ran this custom migration we can consider removing the existing fields in prod db to ensure we dont have duplicated data.